fenix/app/src/main/res/values/static_strings.xml

```diff
@@ -124,4 +123,24 @@
     <string name="translations_bottom_sheet_translate_to">Translate to</string>
     <string name="translations_bottom_sheet_negative_button">Not now</string>
     <string name="translations_bottom_sheet_positive_button">Translate</string>
+
+    <string name="translation_option_bottom_sheet_title">Translation Options</string>
+    <string name="translation_option_bottom_sheet_always_translate">Always offer to translate</string>
+    <string name="translation_option_bottom_sheet_always_translate_in_language">Always translate %1$s</string>
+    <string name="translation_option_bottom_sheet_spanish_language">Spanish</string>
+    <string name="translation_option_bottom_sheet_never_translate_in_language">Never translate %1$s</string>
+    <string name="translation_option_bottom_sheet_never_translate_site">Never translate this site</string>
+    <string name="translation_option_bottom_sheet_translation_settings">Translation settings</string>
+    <string name="translation_option_bottom_sheet_about_translations">About translations in %1$s</string>
+    <string name="accessibility_off_switch">Switch OFF</string>
+    <string name="accessibility_on_switch">Switch ON</string>
```

Comment:
Let's move this out of static strings since content descriptions aren't blocked on translation.

Reply:
Done